zigate/core.py

```diff
@@ -57,7 +57,7 @@ def __getattr__(self, *args, **kwargs):
 BIND_REPORT = True  # automatically bind and report state for light
 SLEEP_INTERVAL = 0.1
 ACTIONS = {}
-WAIT_TIMEOUT = 3
+WAIT_TIMEOUT = 10
```

Comment:
I increased this to avoid too much `No response waiting command` warning messages.

This could probably be expose as a parameter.

Reply:
Note: when invoked from the web UI, this can take too long and cause websocket timeouts.

For now, I've work this around with a cron automation:

```
- id: zigate_refresh_network_map
  alias: 'Zigate: Refresh Zigbee network map every 5 minutes'
  description: ''
  trigger:
  - minutes: /5
    platform: time_pattern
    seconds: '0'
  condition: []
  action:
  - data:
      force: true
    service: zigate.build_network_table
```